### DISCUSSION TITLE
Get users in oneview with powershell

### DISCUSSION THREAD

**Discussion by AutomateIT1979** [HewlettPackard/POSH-HPOneView#671] [Q&A]
Hello,
I'm trying to get all users details with their group details configured in HPE oneview using powershell unfortunately not able to get what's required.
Is there a specific way to get these details?

Need name, directory, type, role and scope

Thank you in advance.

**Comment by VincentBerger** [comment]
In OneView you have local users and directory users. You can get local users with Get-OVUser, that will give you the name, role and scope (in the Permissions property). For directory users, OneView only has a mapping between a directory group and permissions (role and scope) that you can get with Get-OVLdapGroup. OneView does not have a list of individual directory users, for that you need to query the directory server directly.

**Reply by AutomateIT1979** [reply]
Thank you for your time and reply. I will give it a try and keep you posted.
